### PR TITLE
feat: add default settings for the -y flag

### PR DESCRIPTION
**Pull request by Allyedge**
Added default settings for the `-y` flag.

### CODE REVIEW

**Review by EvolutionX-10 — CHANGES_REQUESTED** [review]
it shouldn't ask these except maybe project name is fine
image
npm is default, git is initialised by default

**Comment by Allyedge** [comment]
> it shouldn't ask these except maybe project name is fine image npm is default, git is initialised by default

Alright, working on it.

**Review by jacoobes — DISMISSED** [review]

File: src/commands/init.js [hunk 1]
```diff
@@ -35,16 +35,46 @@ export async function init({ flags }) {
...
+			name: projectName.name,
+			lang: 'typescript',
+			main_dir: 'src',
+			cmds_dir: 'commands',
+			default_prefix: '!',
+		};
+	}
 
-	if (Object.keys(data).length < 5) process.exit(1);
+	if (!isDefault && Object.keys(data).length < 5) process.exit(1);
```
Comment (EvolutionX-10) [inline src/commands/init.js:62]: why `!isDefault` is needed?
even when the defaults are used, data object is getting all 5 properties needed

File: src/commands/init.js [hunk 2]
```diff
@@ -35,16 +35,46 @@ export async function init({ flags }) {
...
 
-	if (!git_init.gitinit) {
-		console.log(`\nAlright\n`);
+	if (!isDefault) {
+		git_init = await prompt([gitInit]);
+	} else {
+		git_init = {
+			gitInit: true,
+		};
+	}
```
Comment (EvolutionX-10) [inline src/commands/init.js:74]:
```suggestion
	isDefault ? git_init.gitInit = true : git_init = await prompt([gitInit])
```
how about this

File: src/commands/init.js [hunk 3]
```diff
@@ -66,7 +96,13 @@ export async function init({ flags }) {
...
 	}
 
-	const pm = await npm();
+	let pm;
+
+	if (!isDefault) {
+		pm = await npm();
+	} else {
+		pm = 'npm';
+	}
```
Comment (EvolutionX-10) [inline src/commands/init.js:105]:
```suggestion
	let pm;
	isDefault ? pm = 'npm' : await npm()
```

**Review by EvolutionX-10 — CHANGES_REQUESTED** [review]

**Comment by Allyedge** [comment]
Looks good.

**Review by EvolutionX-10 — APPROVED** [review]

**Pull request merged**